snowfall/training/ctc_graph.py

```diff
@@ -15,8 +15,8 @@ def build_ctc_topo(tokens: List[int]) -> k2.Fsa:
     symbols to a single output symbol.
 
     Caution:
-      The resulting topo is an FST. Epsilons are on the left
-      side (i.e., ilabels) and tokens are on the right side (i.e., olabels)
+      The resulting topo is an FST. Epsilons are on the right
```

Comment:
It would be better if we can sync the documentation with PR https://github.com/k2-fsa/k2/pull/602/files where @danpovey and I left some comments and @csukuangfj has fixed them.

Reply:
Sure, I will modify it when @csukuangfj's PR is ready.